EvidenceApi/V1/Gateways/NotifyGateway.cs

```diff
@@ -170,5 +172,22 @@ private DocumentType GetDocumentType(EvidenceRequest evidenceRequest)
 
         private string MagicLinkFor(EvidenceRequest evidenceRequest) => $"{_options.EvidenceRequestClientUrl}resident/{evidenceRequest.Id}";
         private string ResidentPageLinkFor(EvidenceRequest evidenceRequest) => $"{_options.EvidenceRequestClientUrl}teams/{_documentTypeGateway.GetTeamIdByTeamName(evidenceRequest.Team)}/dashboard/residents/{evidenceRequest.ResidentId}";
+
+        private string FormatDocumentTypes(EvidenceRequest evidenceRequest)
+        {
+            var teamDocumentTypes = _documentTypeGateway.GetDocumentTypesByTeamName(evidenceRequest.Team);
+            var evidenceRequestDocumentTypes = evidenceRequest.DocumentTypes;
+            var documentTypeTitles = new List<string>();
+
+            foreach (var documentType in teamDocumentTypes)
+            {
+                if (evidenceRequestDocumentTypes.Contains(documentType.Id))
+                {
+                    documentTypeTitles.Add(documentType.Title);
+                }
+            }
+
+            return string.Join(",\n", documentTypeTitles);
+        }
```

Comment:
Is this function needed? I think the same result can be achieved by doing this:
`var documentTypesTitles = evidenceRequest.DocumentTypes.ConvertAll<string>(dt => _documentTypeGateway.GetDocumentTypeByTeamNameAndDocumentTypeId(evidenceRequest.Team, dt).Title);`

And then:
`string.Join(",\n", documentTypesTitles);`
when we pass the parameter to the GovNotify template

Reply:
Ah yeah I think that would do the same thing -- I'll try it out.

Reply:
I've changed `ConvertAll` to `Select` as `.Title` is already a string.

Reply:
Spoke to Bogdan about this change, as it requires n amount of gateway calls, instead of one. Currently, the gateway is calling the file system, so not that computationally expensive, but could be if we moved this functionality away to e.g. a db.

I've refactored the `foreach` loop to 
`var documentTypeTitles = teamDocumentTypes.Where(tdt => evidenceRequestDocumentTypes.Contains(tdt.Id)).Select(dt => dt.Title);`